### PR TITLE
feat: adding node example to sdk

### PR DESCRIPTION
**Pull request by Abhikumar98**

### CODE REVIEW

**Comment by Abhikumar98** [comment]
@chybisov lmk if I should remove the yarn.lock file

File: examples/node/index.js [hunk 1]
```diff
@@ -13,28 +13,35 @@ async function demo() {
...
+    fromTokenAddress: findDefaultToken(CoinKey.USDC, ChainId.AVA).address,
+    toChainId: ChainId.AVA, // Avalanche
+    toTokenAddress: findDefaultToken(CoinKey.USDT, ChainId.AVA).address,
     options: {
       slippage: 0.03, // = 3%
       allowSwitchChain: false, // execute all transaction on starting chain
-      exchanges: {
-        allow: [], // only find direct transfers
-      },
+      // exchanges: {
```
Comment (Addminus) [inline examples/node/index.js:42]: do we want to remove these comments?

File: examples/node/index.ts [hunk 2]
```diff
@@ -21,46 +25,49 @@ async function demo() {
...
+    fromTokenAddress: findDefaultToken(CoinKey.USDC, ChainId.AVA).address,
+    toChainId: ChainId.AVA, // Avalanche
+    toTokenAddress: findDefaultToken(CoinKey.USDT, ChainId.AVA).address,
     options: {
       slippage: 0.03, // = 3%
       allowSwitchChain: false, // execute all transaction on starting chain
-      exchanges: {
-        allow: [], // only find direct transfers
-      },
+      // exchanges: {
```
Comment (Addminus) [inline examples/node/index.ts:55]: same here?

File: examples/node/index.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { LiFi } from '@lifi/sdk'
+import { LiFi, ChainId, CoinKey, findDefaultToken } from '@lifi/sdk'
+import { RPCProvider } from '@lifi/rpc-wrapper'
+import { Wallet } from 'ethers'
 
-import ethers from 'ethers'
-
-const lifis = new LiFi()
+const lifi = new LiFi()
```
Comment (chybisov) [inline examples/node/index.js:5]: We should also initialize LiFi in the JS example with the required integrator string to show the users it is always needed.

File: examples/node/package.json [hunk 4]
```diff
@@ -7,11 +7,19 @@
...
+    "start": "ts-node index.ts"
   },
   "devDependencies": {
     "@types/node": "^18.11.17",
     "ts-node": "^10.8.2",
     "typescript": "^4.9.4"
+  },
+  "resolutions": {
+    "@lifi/sdk": "portal:/Users/abhishekkumar/Desktop/work/sdk"
   }
```
Comment (chybisov) [inline examples/node/package.json:24]: Please remove this once you finish testing 🙂

**Review by chybisov — CHANGES_REQUESTED** [review]

File: examples/node/index.ts [hunk 5]
```diff
@@ -21,46 +25,49 @@ async function demo() {
...
-    apiUrl: 'https://li.quest', // DEFAULT production endpoint
-    rpcs: {
-      // You can provide custom RPCs
-      137: ['https://polygon-rpc.com/'],
-    },
-    multicallAddresses: {
-      // You can provide custom addresses for multicall
-      137: '0x02817C1e3543c2d908a590F5dB6bc97f933dB4BD',
-    },
+    integrator: 'lifi-sdk', // DEFAULT 'lifi-sdk'
```
Comment (Addminus) [inline examples/node/index.ts:65]: can we find a better integrator string? "lifi-sdk-sample" for example?

**Review by Addminus — CHANGES_REQUESTED** [review]

File: examples/node/index.ts (hunk 5 above)
Comment (Addminus) [reply examples/node/index.ts:65]: just something that hints that the request comes from the example

Comment (Addminus) [reply examples/node/index.ts:65]: also the js file has no integrator?

File: examples/node/package.json (hunk 4 above)
Comment (Abhikumar98) [reply examples/node/package.json:24]: Done, removed it.

File: examples/node/index.js (hunk 3 above)
Comment (Abhikumar98) [reply examples/node/index.js]: Yes, I made the changes already but the push was pending 😄

File: examples/node/index.js (hunk 1 above)
Comment (Abhikumar98) [reply examples/node/index.js]: removed

File: examples/node/index.ts (hunk 2 above)
Comment (Abhikumar98) [reply examples/node/index.ts]: removed

File: examples/node/index.ts (hunk 5 above)
Comment (Abhikumar98) [reply examples/node/index.ts]: Added, it was present already but the push failed.

**Comment by chybisov** [comment]
Please check the latest updates in this branch https://github.com/lifinance/sdk-node-sample/tree/contract-calls and update this one with contract calls example 🙂

**Review by chybisov — APPROVED** [review]

**Pull request merged**